spec/migrations/20160413191759_correct_sti_type_on_cloud_resource_quota_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_migration
+
+describe CorrectStiTypeOnCloudResourceQuota do
+  let(:cloud_resource_quota_stub) { migration_stub(:CloudResourceQuota) }
+  let(:crq_entries) do
+    [
+      {:old => {:type => 'somethingelse'},
+       :new => {:type => 'somethingelse'}},
+      {:old => {:type => 'CloudResourceQuotaOpenstack'},
+       :new => {:type => 'ManageIQ::Providers::Openstack::CloudManager::CloudResourceQuota'}},
+      {:old => {:type => 'OpenstackResourceQuota'},
+       :new => {:type => 'ManageIQ::Providers::Openstack::CloudManager::CloudResourceQuota'}},
+    ]
+  end
+
+  migration_context :up do
+    it 'migrates a series of representative rows' do
+      crq_entries.each do |x|
+        x[:up] = cloud_resource_quota_stub.create!(x[:old])
+      end
+
+      migrate
+
+      crq_entries.each do |x|
+        expect(x[:up].reload).to have_attributes(:type => x[:new][:type])
```

Comment:
`have_attributes`  can be you used for more attributes.
in this case it is enough 
```
       expect(x[:up].reload.type).to eq(x[:new][:type])
```
it is shorter, what do you think?